prime-router/src/test/kotlin/CsvFileTests.kt

```diff
@@ -92,15 +105,35 @@ class CsvFileTests {
     private fun loadTestOrganizations(settings: FileSettings) {
         val loadingStream = File(inputPath + "test-organizations.yml").inputStream()
         settings.loadOrganizations(loadingStream)
-        assertEquals(2, settings.receivers.size)
-        assertEquals(2, settings.findReceiver("federal-test.receiver")?.jurisdictionalFilter?.size)
+        assertThat(settings.receivers).hasSize(2)
+        assertThat(
+            settings.findReceiver("federal-test.receiver")?.jurisdictionalFilter
+        ).isNotNull().hasSize(2)
     }
 
     private fun loadTestSchemas(metadata: Metadata) {
         metadata.loadSchemaCatalog(inputPath)
         val schema = metadata.findSchema(defaultSchema)
-        assertNotNull(schema)
-        assertEquals(7, schema.elements.size)
-        assertEquals("lab", schema.elements[0].name)
+        assertThat(schema).isNotNull()
+        assertThat(schema!!.elements).hasSize(7)
+        assertThat(schema.elements[0].name).isEqualTo("lab")
+    }
+
+    companion object {
+        private fun Assert<ReadResult>.hasNoWarnings(): Assert<ReadResult> = transform { actual ->
+            if (actual.warnings.count() == 0) {
+                actual
+            } else {
+                expected("expected: ReadResult to have no warnings, but it had ${show(actual.warnings.count())}")
+            }
+        }
+
+        private fun Assert<ReadResult>.hasNoErrors(): Assert<ReadResult> = transform { actual ->
+            if (actual.errors.count() == 0) {
+                actual
+            } else {
+                expected("expected: ReadResult to have no errors, but it had ${show(actual.errors.count())}")
+            }
+        }
```

Comment:
Here are two extension methods that takes in `ReadResult` and then returns it back allowing the assertions to be chained.